### ISSUE TITLE
Application updates only once

### ISSUE THREAD

**Issue by taaoo75** [MicrosoftDocs/windows-uwp#1066]
The promise was that once installed, the app will always keep itself current and keep on updating whenever there is a newer version detected at the source location. This does not happen but once. When installed, the app updates itself to the next version that is available at the source location and stops doing any updates after that. This issue has been long standing since the release in April but so far nobody has informed as to when, if at all, this will be fixed. With this functionality broken, the whole use case of side loading an app is pretty much invalid.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3eebf940-9282-41a2-f236-55984ff76aeb
* Version Independent ID: a1b3c572-c6ba-eff7-637d-cbcb20801c57
* Content: [Create an App Installer file with Visual Studio - UWP app developer](https://docs.microsoft.com/en-us/windows/uwp/packaging/create-appinstallerfile-vs)
* Content Source: [windows-apps-src/packaging/create-appinstallerfile-vs.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/packaging/create-appinstallerfile-vs.md)
* Product: **windows**
* GitHub Login: @ridomin
* Microsoft Alias: **rmpablos**

**Comment by ridomin** [comment]
Hi, I've not been able to reproduce, can you provide additional details?
Which OS? Which SDK are you targeting? Which Update Policy are you using? Any specifics about your app?

**Comment by taaoo75** [comment]
Thanks for the kind and prompt reply my friend. Here are the particulars you requested.

Windows Specification
 
Edition: windows 10Pro.
Version: 1803
OS build: 17134.345
 
.Net framework version: 4.7
VS version: 15.7.1, Professional 2017
Windows 10 SDK (10.0.16299.0) for UWP

**Comment by rido-min** [comment]
@taaoo75 Visual Studio generates the app installer file when Target Platform Min Version is 10.0.17134.0 or greater, I hope you have a most recent SDK version.

Also, will be good to know which kind of app (C#, C++, JS UWP or Desktop Bridge), and where are you hosting your files (HTTP, Shared Folder). 

If you could share the original appinstaller file as well the updated one will help to understand the issue. If you want do not want to share it here you can contact me at rmpablos at microsoft dot com.


Thanks,
Rido

**Comment by taaoo75** [comment]
Rido my friend,

Tried sending you the files as a zip package at rmaplos dot Microsoft dot com but the email got rejected by the host 🙁
________________________________

**Comment by rido-min** [comment]
@taaoo75  sorry, there is a typo in the email (corrected). It should be rmpablos

**Comment by rido-min** [comment]
This is a known bug in 1803, it has been fixed in the October 2018 update aka 1809

**Comment by mhmd-azeez** [comment]
@rido-min From my experience, the bug is still present.

**Comment by nonasi** [comment]
If you have an app & appinstaller file that repro it - can you point me to them?

**Comment by mhmd-azeez** [comment]
@nonasi I tried it out again, and it seems that it works now. The first time the only the client computer was updated to 1809, but now that I have also updated my dev machine, it seems to work.

**Comment by kEph13** [comment]
I'm not sure if I'm seeing the same "only once" bug as others, but here's my experience:
-I copied the files over to the target machine (running 1809 LTSC) and installed, with the update setting on "check on boot"
-I put a new version on the web server, and it updated properly
-I put another new version on the web server (with no code changes but the version, and with the update setting changed to every hour), and it failed to update
-I tried to update manually by navigating to the web server page and clicking install, and got "App installation failed with error message: SQLITE_CONSTRAINT_UNIQUE (0x87af0813)"
-I then removed the app and tried reinstalling, both locally and from the web server location, and continue to get that message
-If I run Add-AppxPackage manually it works, but I continue to get the same error now any time I use the app installer.

I did have to download the App Installer appx from the store and add it to the machine, since it's LTSC and doesn't have the store available. Is there any plan for supporting this whole framework on LTSC in a cleaner way? It would be a great fit for kiosk apps like ours, where stability is important but we still want to be able to push updates occasionally.
